app/components/edit-text/template.hbs

```diff
@@ -0,0 +1,11 @@
+	{{#if isTitle}}
```

Comment:
I'm wondering if we should separate this title edit into its own component. Probably not necessary. What are your thoughts on this?